apps/server/src/controllers/userControllers/userUpdateKyc.ts

```diff
@@ -0,0 +1,90 @@
+import { Response } from 'express'
+import { RequestExt } from 'middleware/authorization/authentication'
+import { editProfileSchema } from '../../utils/validators'
+import { HTTP_STATUS_CODE } from '../../constants'
+import { v2 as cloudinary } from 'cloudinary'
+
+export const updateKycProfile = async (req: RequestExt, res: Response) => {
+  try {
+    const { _user: user, ...rest } = req.body
+
+    const validationResult = editProfileSchema.safeParse(rest)
+
+    if (!validationResult.success) {
+      return res.status(HTTP_STATUS_CODE.BAD_REQUEST).json({
+        message: 'Validation error',
+        issues: validationResult.error.issues,
+      })
+    }
+
+    const {
+      firstName,
+      lastName,
+      email,
+      phone,
+      gender,
+      occupation,
+      date_of_birth,
+      bvn,
+      address,
+      identification_type,
+      identification_number,
+      identification_doc,
+      proof_of_address_doc,
+    } = validationResult.data
+
+    const files = req.files as {
+      [fieldname: string]: Express.Multer.File[]
+    }
+    console.log(files)
```

Comment:
You can also remove the logged files

Reply:
I thought I've removed this already. Okay, will do so.

Reply:
Oh, now I remember. I've already done that in the updateUserKyc branch. I also refactored some few codes in the branch. So I think when they are all merged, it'll be fixed.